### PR TITLE
Update indexing.rst

### PR DESCRIPTION
**Pull request by fgrcon**
Fixes: #
-  fix description for meta data column getIcon [fgrcon]

Improves:

Changes proposed in this pull request:
## 
## 
##

### CODE REVIEW

File: develop/plone/searching_and_indexing/indexing.rst [hunk 1]
```diff
@@ -384,7 +384,7 @@ Some interesting columns
 
 * getRemoteURL: Where to go when the object is clicked
 
-* getIcon: Which content type icon is used for this object in the navigation
+* getIcon: Since Plone 5.02 - Boolean value which is set to True, when item has or is an image (used for showing thumbs in lists, portlets, etc. ). Content-type-icons (aka portaltype-icons) ( e.g.: for folder, document, news item etc.) are rendered as fontello fonts since Plone 5.0.
```
Comment (svx) [inline develop/plone/searching_and_indexing/indexing.rst]: - it should be 5.0.2 right ?
- we should use :guilabel: to highlight the word 'True'
- we should use (versionadded)[http://www.sphinx-doc.org/en/stable/markup/para.html] to 'highlight' the different Plone versions

**Comment by svx** [comment]
Thanks for that !!!!

 Please see my comments :)

File: develop/plone/searching_and_indexing/indexing.rst (hunk 1 above)
Comment (fgrcon) [inline develop/plone/searching_and_indexing/indexing.rst]: @svx: thanks for your hints. I tried to use .. versionchanged:: ... but all I managed was to ruin the preview - Im to silly for sphinx markup?

**Comment by svx** [comment]
don't worry I will change it later :) 

I will merge for now, thanks for your contribution !

**Pull request merged**